compatibility/basic-attestation-on-localhost-api-version-bump/test.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+[ -n "${RUST_KEYLIME_UPSTREAM_URL}" ] || RUST_KEYLIME_UPSTREAM_URL="https://github.com/keylime/rust-keylime.git"
+[ -n "${RUST_KEYLIME_UPSTREAM_BRANCH}" ] || RUST_KEYLIME_UPSTREAM_BRANCH="master"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+
+        # update /etc/keylime.conf
+        limeBackupConfig
+        rlRun "limeUpdateConf logger_root level DEBUG"
+        rlRun "limeUpdateConf logger_keylime level DEBUG"
+        rlRun "limeUpdateConf handler_consoleHandler level DEBUG"
+        # verifier
+        rlRun "limeUpdateConf revocations enabled_revocation_notifications '[]'"
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        # agent
+        rlRun "limeUpdateConf agent enable_revocation_notifications false"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        # start keylime_verifier
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        # create allowlist and excludelist
+        rlRun "limeCreateTestPolicy ${TESTDIR}/*"
+
+        WORKDIR=$( mktemp -d -p "/var/tmp" )
+    rlPhaseEnd
+
+    rlPhaseStartTest "Compile old and new keylime agent"
+        rlRun "git clone ${RUST_KEYLIME_UPSTREAM_URL} ${WORKDIR}/rust-keylime"
+        rlRun "pushd ${WORKDIR}/rust-keylime"
+        rlRun "git checkout v0.2.1"
+        rlRun "cargo build && cp ./target/debug/keylime_agent ${WORKDIR}/keylime_agent-2.1"
+        rlRun "git checkout ${RUST_KEYLIME_UPSTREAM_BRANCH}"
```

Comment:
Is it necessary to build the "current" version again? Agent built from PR sources should be already built by the former `/setup/` task so we should simply use it, or not?

Reply:
I wasn't sure if the current version would be always available. It would be great if we can avoid recompiling it again.

Reply:
There will be one installed, otherwise all tests would fail.